helpers/guilds/modifyGuildMfaLevel.ts

```diff
@@ -0,0 +1,12 @@
+import type { Bot } from "../../mod.ts";
+import type { MfaLevels } from "../../types/shared.ts";
+
+/** Modify a guild's MFA level. Requires guild ownership. */
+export async function modifyGuildMfaLevel(bot: Bot, guildId: bigint, mfaLevel: MfaLevels, reason?: string) {
+  return await bot.rest.runMethod<MfaLevels>(
```

Comment:
```suggestion
  return await bot.rest.runMethod<void>(
```

Reply:
https://discord.com/developers/docs/resources/guild#modify-guild-mfa-level

# Returns the updated [level](https://discord.com/developers/docs/resources/guild#guild-object-mfa-level) on success.

<img width="472" alt="image" src="https://user-images.githubusercontent.com/87189679/188176932-08c8e539-342c-4114-9b14-96aa90a1c91a.png">

Reply:
This is best avoided as it will create confusion, if it is just returning the value that we sent then no point in returning and will make cleaner end user code if this returns void